src/key_info_managers/on_disk_manager/mod.rs

```diff
@@ -33,15 +33,21 @@ use parsec_interface::requests::{AuthType, ProviderId};
 use std::collections::HashMap;
 use std::convert::TryFrom;
 use std::ffi::OsStr;
+use std::fs::Permissions;
 use std::fs::{DirEntry, File};
 use std::io::{Error, ErrorKind, Read, Write};
 use std::ops::Deref;
+use std::os::unix::fs::PermissionsExt;
 use std::path::{Path, PathBuf};
 use std::{fmt, fs};
 
 /// Default path where the mapping files will be stored on disk
 pub const DEFAULT_MAPPINGS_PATH: &str = "/var/lib/parsec/mappings";
 
+///File permissions for all files/directories under database directory
+///Should only be visible to parsec user
+pub const FILE_PERMISSION: u32 = 0o770;
```

Comment:
why would we need +x on files? Shouldn't it be 770 for dirs and 660 for files?

Reply:
You are right. I've updated the permissions now.